### PR TITLE
add events e2e

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: test/pkg/e2e/localpolicy_test.go [hunk 1]
```diff
@@ -154,6 +154,44 @@ var _ = Describe("Apply local policy to the managed clusters", Ordered,
...
+					if err != nil {
+						return err
+					}
+					defer rows.Close()
+
+					// policies, if leahfubname check remove the kv
+					for rows.Next() {
+						columnValues, _ := rows.Values()
+						if len(columnValues) <= 0 {
+							return fmt.Errorf("the event.local_policies record is 0")
```
Comment (yanmxa) [inline test/pkg/e2e/localpolicy_test.go:171]: Can we verify the information of the event, like the policyId and clusterId?

File: test/pkg/e2e/localpolicy_test.go [hunk 2]
```diff
@@ -154,6 +154,44 @@ var _ = Describe("Apply local policy to the managed clusters", Ordered,
...
+				Eventually(func() error {
+					rows, err := postgresConn.Query(context.TODO(),
+						"select policy_id,event_name,leaf_hub_name from event.local_root_policies")
+					if err != nil {
+						return err
+					}
+					defer rows.Close()
+
+					// policies, if leahfubname check remove the kv
+					for rows.Next() {
```
Comment (yanmxa) [inline test/pkg/e2e/localpolicy_test.go:187]: Also, Can we verify the policyid of the event?

File: test/pkg/e2e/localpolicy_test.go (hunk 1 above)
Comment (ldpliu) [reply test/pkg/e2e/localpolicy_test.go]: done

File: test/pkg/e2e/localpolicy_test.go (hunk 2 above)
Comment (ldpliu) [reply test/pkg/e2e/localpolicy_test.go:211]: done

File: test/pkg/e2e/localpolicy_test.go [hunk 3]
```diff
@@ -129,31 +134,98 @@ var _ = Describe("Apply local policy to the managed clusters", Ordered,
...
+						policyId, cluster_id, leafhub := "", "", ""
+						if err := rows.Scan(&policyId, &cluster_id, &leafhub); err != nil {
+							return err
+						}
+						if policyIds.Has(policyId) {
+							managedClusterIds.Insert(cluster_id)
+							currentPolicyIds.Insert(policyId)
+						}
+					}
+					if managedClusterIds.Len() != len(managedClusters) {
```
Comment (yanmxa) [inline test/pkg/e2e/localpolicy_test.go:191]: This line seems duplicated with the line 158. Maybe the `managedClusterIds` should be a local variable?

File: test/pkg/e2e/localpolicy_test.go [hunk 4]
```diff
@@ -129,31 +134,98 @@ var _ = Describe("Apply local policy to the managed clusters", Ordered,
...
+				By("Verify the local policy events is synchronized to the global hub event.local_policies table")
+				Eventually(func() error {
+					rows, err := postgresConn.Query(context.TODO(),
+						"select policy_id,cluster_id,leaf_hub_name from event.local_policies")
+					if err != nil {
+						return err
+					}
+					defer rows.Close()
+					currentPolicyIds := sets.NewString()
+
```
Comment (yanmxa) [inline test/pkg/e2e/localpolicy_test.go:176]: currentManagedClusterIds?

Comment (ldpliu) [reply test/pkg/e2e/localpolicy_test.go]: Thanks 
Fixed.

File: test/pkg/e2e/localpolicy_test.go (hunk 3 above)
Comment (ldpliu) [reply test/pkg/e2e/localpolicy_test.go:191]: Thanks 
Fixed.

**Pull request merged**